src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -377,6 +379,14 @@ class DocBaseViewer extends BaseViewer {
         this.showPreload();
 
         const template = this.options.representation.content.url_template;
+        const { extension } = this.options.file;
```

Comment:
Is `this.options.file` always defined as an object? Should we move this logic to DocumentViewer?

Reply:
Seems like it is always passed into the viewer options: https://github.com/box/box-content-preview/blob/19722e68d00a62b588ef06f7fe824a55afa0e107/src/lib/Preview.js#L1211